src/_includes/head.html

```diff
@@ -111,6 +111,13 @@
     ga('create', '{% t analytics.google-id %}', 'codurance.com');
     ga('set', 'forceSSL', true);
 
+    // code for installing Google Tag Manager
+    (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-WMKBC5G');
```

Comment:
We should put this in the config file as the google-id for analitics
`analytics.google-id`